index.txt

```diff
@@ -162,13 +170,14 @@ install_caddy()
 
 	echo "Putting caddy in $install_path (may require password)"
 	$sudo_cmd mv "$PREFIX/tmp/$caddy_bin" "$install_path/$caddy_bin"
+	$sudo_cmd setcap cap_net_bind_service=+ep "$install_path/$caddy_bin"
```

Comment:
Setcap may not be available on some systems. We should make sure it exists before running this. (e.g. doesn't exist on Mac)